### DISCUSSION TITLE
How to do VSCode for Winui3 projects

### DISCUSSION THREAD

**Discussion by nczsl** [microsoft/microsoft-ui-xaml#10218] [General]
I don't see any official documentation on how to use VSCode to develop Winui3; So what I want to ask is, how should I develop Winui3 applications using VSCode and VNet (. net 9)?

**Comment by jamespack** [comment]
While probably not impossible, it's highly doubtful that would be enjoyable. Why not visual studio?

**Comment by Perxenic-Acid** [comment]
What I know is, there is not ANY extension for WinUI3 projs in VSCode's extension store yet. Please use Visual Studio instead whose Coder.
